superset-frontend/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "superset",
-  "version": "2.0.0",
+  "version": "0.0.0-dev",
```

@AAfghahi it looks like this changed
```suggestion
  "version": "2.0.1",
```